platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/NetworkUtils.java

```diff
@@ -171,4 +186,36 @@ public static String formatException(final Throwable e) {
             throw new PlatformConstructionException("A problem occurred while creating the SocketFactory", e);
         }
     }
+
+    /**
+     * identifies the client on the other end of the socket using their signing certificate.
+     *
+     * @param certs        the list of all certificates in the public keystore
```

Comment:
this part of the javadoc is wrong, this is a the certificate chain presented my the peer through TLS

Reply:
updated. Moved it all into a separate class .